lib/rouge/lexers/datastudio.rb

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    class Datastudio < RegexLexer
+      tag 'datastudio'
+      filenames '*.job'
+      mimetypes 'text/x-datastudio'
+
+      title "Datastudio"
+      desc 'Datastudio scripting language'
```

Comment:
Is there an official website for this language?

Reply:
No, there is no language website. I'm in the process of writing the language documentation website, which is distributed with the Datastudio product itself.